applications/launchpad/backend/src/api/mod.rs

```diff
@@ -55,3 +59,31 @@ pub async fn health_check(image: &str) -> String {
         Err(_err) => format!("image {} not found", image),
     }
 }
+
+#[tauri::command]
+pub async fn wallet_events(app: AppHandle<Wry>) -> Result<(), String> {
+    info!("Setting up event stream");
+    let mut wallet_client = GrpcWalletClient::new();
+    let mut stream = wallet_client.stream().await.map_err(|e| e.chained_message()).unwrap();
+    let app_clone = app.clone();
+    tauri::async_runtime::spawn(async move {
+        while let Some(response) = stream.next().await {
+            let value = response.transaction.unwrap();
```

Comment:
Can this fail?
- If not, please put a comment in to indicate this.
- If so, handle the error and return gracefully.

Reply:
This is a bug and I will fix it.